pkgs/top-level/stage.nix

```diff
@@ -247,7 +247,10 @@ let
             musleabi = lib.systems.parse.abis.musleabi;
             musleabihf = lib.systems.parse.abis.musleabihf;
           }.${stdenv.hostPlatform.parsed.abi.name}
-            or lib.systems.parse.abis.musl;
+            or (if      stdenv.hostPlatform.isMips64n32 then lib.systems.parse.abis.muslabin32
+                # commented out because glibc does not yet understand *-muslabi64
+                #else if stdenv.hostPlatform.isMips64n64 then lib.systems.parse.abis.muslabi64
+                else lib.systems.parse.abis.musl);
```

Comment:
With the above, is this still needed?

Reply:
testing now...

Reply:
Regarding the commented-out line, and the comment above it: I put that in before making the change on this line:

https://github.com/NixOS/nixpkgs/pull/161158/files#diff-45dcd931b43fdf555b6630a30b84f99b8f86a3791a947598084a74216b68c639R24

... which apparently fixed the issue.